controller/array_action/errors.py

```diff
@@ -203,3 +205,9 @@ class InvalidCliResponseError(BaseArrayActionException):
 
     def __init__(self, details):
         self.message = messages.InvalidCliResponseError_message.format(details)
+
+
+class FlashcopyCreationError(BaseArrayActionException):
```

Comment:
I think `Flashcopy` is not a cross-storage term.
and we usually let unexpected "type of exceptions" to be excepted by the controller server as an internal excpetion, so no need for this (same for [deletion](https://github.com/IBM/ibm-block-csi-driver/pull/165/files#diff-410ba176065c9dbbe7a1d72c4e842eb0R216))

Reply:
I think it necessary that it will stay, there is no similar exception for that and its important output.

Reply:
why is it an important output?
how would it look without it?

Reply:
done